packages/api/lib/rulesHelpers.js

```diff
@@ -591,7 +591,6 @@ async function updateRuleTrigger(original, updates, knex) {
   }
   case 'kinesis':
     if (valueUpdated) {
-      await deleteKinesisEventSources(knex, mergedRule);
```

Comment:
This also didn't match main. Changes were made to the model but not the helper (which is a duplicate of the model method). This ports that change to the helper.

Reply:
Right. That's a better way to think of it. In main, `deleteKinesisEventSources` was moved from the Rule model to the endpoint (which calls `updateRuleTrigger`). Now that we're using the rulesHelpers instead of the model, the same change needed to be applied to the helper.

Reply:
Hmm, I'm not sure I'm clear on this then - the deletion above looks like it's deleting the merged rule, but the original code in main and your update here looks like it's deleting the 'oldRule' -- is it just a bug in P3 that this isn't

```
await deleteKinesisEventSources(knex, original);
```
?

Reply:
re: this in huddle code review - this *also* matches main and appears to be correct - removing the existing rules shouldn't matter as we're re-adding them, unless the downstream update methods aren't idempotent, which they appear to be.